packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -70,6 +72,17 @@ const mockFlags: { [key: string]: any } = {
   getBlockByNumberValue: null,
 };
 
+const SEPOLIA = {
+  chainId: toHex(11155111),
+  type: NetworkType.sepolia,
+  ticker: NetworksTicker.sepolia,
+};
+const GOERLI = {
+  chainId: toHex(5),
+  type: NetworkType.goerli,
+  ticker: NetworksTicker.goerli,
+};
```

Comment:
looks like we only use chainId from these. Maybe we use `BUILT_IN_NETWORKS` from NetworkController or `ChainId` from controller-utils instead

Reply:
working on this

Reply:
https://github.com/MetaMask/core/pull/3643/commits/81fcd605873eb74e6f05d47cb9f85a46a4d2d216